### DISCUSSION TITLE
How to Become a Hound FM Curator

### DISCUSSION THREAD

**Discussion by exarchist** [Hound-fm/community-guidelines#5] [Q&A]
The link here https://github.com/Hound-fm/community-guidelines discusses a system whereby a Curator finds content and discovers it for Hound FM, however the details on how to become a curator are lacking.

Additional detail on how to discover content would also be appreciated. The impression that I have is simply using the LBRY "repost" feature is enough to trigger a discovery, however the instructions on this are not clearly written.

**Comment by btzr-io** [comment]
> Additional detail on how to discover content would also be appreciated. The impression that I have is simply using the LBRY "repost" feature is enough to trigger a discovery, however the instructions on this are not clearly written.

Yes this was the old system,  the idea was to analyze / store music that people discover  using any lbry "platform". However at the time there where lots of limitations ( there still are some ) so the only interaction that was possible to track was "reposts".
I removed this "requirement" because the lack of "curators" ( users reposting content ).

**Reply by exarchist** [reply]
Is the system of "curators" no longer valid then? There are many podcasts and musicians on LBRY/Odysee, but very few I found on Hound.FM.

**Reply by btzr-io** [reply]
> Is the system of "curators" no longer valid then?

There is no longer a curation system. I can try to revive it there is enough "curators" interested. 

>  There are many podcasts and musicians on LBRY/Odysee, but very few I found on Hound.FM.

Yes most podcast / musicians upload videos instead of audio. Nothing can be done about. Also they don't use the proper metadata like tags to identify what the channel / content is about. Also keep in mind that most of the music or even podcasts are just reuploads ( copyrighted content ).

**Reply by btzr-io** [reply]
Now there are different strategies to analyze channels / content before automatically store it in the elasticsearch database. 

But there is also a list to manually add content if you are really interested on adding channels: 
https://github.com/Hound-fm/podcatcher/blob/main/data/safe_list.json

This doesn't guaranty that the content will be added, since the channel needs to have "valid content" ( audio ) and also original content only otherwise it will be removed.

**Reply by exarchist** [reply]
> This doesn't guaranty that the content will be added, since the channel needs to have "valid content" ( audio ) and also original content only otherwise it will be removed.

I see, so channels with music videos are not valid for aggregation because their content is not audio-only?

**Reply by btzr-io** [reply]
Channels can have any kind of content as long as they have at least one audio stream / claim ( A song or podcast episode )
